### PR TITLE
Add VITE token

### PR DESCRIPTION
**Pull request by TerrAcoHee**
VITE token is a multi-chain token that is trading on ETH, BSC & VITE, I have linked the list of tokens here for your quick reference.

VITE has been listed on Binance, okex, wazirx, bittrex, pancake, coinex

Contract audit Report: https://vite.org/audit/Audit-Report-VITE-BEP20-v1.0.pdf

You can also find more info at our official website: https://vite.org/token

Please let me know if you need any further information to approve this. thanks

### CODE REVIEW

**Comment by ramsty** [comment]
Hi,
Your logo should be a full screen, please change.
Example:
137116359-c2bb87b7-239a-4c8d-b54e-4a679c621186

**Comment by BlackeyHou** [comment]
Hi @ramsty I have changed the logo, please check, thanks

File: blockchains/smartchain/assets/0x2794DAD4077602eD25A88d03781528D1637898B4/info.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "VITE Token",
+    "website": "https://vite.org",
+    "description": "VITE(Vite) is a lightning-fast blockchain where transactions incur zero fees.",
+    "explorer": "https://bscscan.com/token/0x2794DAD4077602eD25A88d03781528D1637898B4",
+    "type": "BEP20",
+    "symbol": "VITE",
+    "decimals": 18,
+    "status": "active",
+    "id": "0x2794DAD4077602eD25A88d03781528D1637898B4",
```
Comment (aminpsycho) [inline blockchains/smartchain/assets/0x2794DAD4077602eD25A88d03781528D1637898B4/info.json:10]: ۰

**Review by aminpsycho — APPROVED** [review]
۰

**Pull request merged**